xml/src/main/scala/fs2/data/xml/package.scala

```diff
@@ -103,15 +104,10 @@ package object xml {
       *
       * This pipe can be used when whitespace characters are not relevant to the application
       * and to make it more readable to human beings.
-      *
-      * @param collapseEmpty Whether empty tags are collapsed in a single self closing tag
-      * @param indent THe indentation string
-      * @param attributeThreshold Number of attributes above which each attribute is rendered on a new line
       */
-    def pretty[F[_]](collapseEmpty: Boolean = true,
```

Comment:
Aren't you breaking bincompat here? Looks to me like this is public, so `pretty` would need to be kept and delegate to `prettyPrint`. Also, `attributeThreshold` is gone in favour of the more general `width`, right?

Reply:
You can't break compat if things are not released yet 😆 The pretty thing in XML is not released, I just merged it a few weeks ago.

Reply:
Fair point 😄